ee/clickhouse/queries/experiments/funnel_experiment_result.py

```diff
@@ -60,10 +61,20 @@ def __init__(
         breakdown_key = f"$feature/{feature_flag.key}"
         self.variants = [variant["key"] for variant in feature_flag.variants]
 
+        # our filters assume that the given time ranges are in the project timezone.
```

Comment:
@EDsCODE just FYI, new awkwardness because of the project timezone handling 😅 

Also see: https://posthog.slack.com/archives/C034XD440RK/p1665751823853749